lightkurve/tests/test_lightcurve.py

```diff
@@ -56,8 +56,15 @@ def test_bitmasking(quality_bitmask, answer):
 def test_lightcurve_fold():
     """Test the ``LightCurve.fold()`` method."""
     lc = LightCurve(time=[1, 2, 3], flux=[1, 1, 1])
-    assert_almost_equal(lc.fold(period=1).time[0], 0)
-    assert_almost_equal(lc.fold(period=1, phase=-0.1).time[0], 0.1)
+    assert_almost_equal(lc.fold(period=1).time[0], -0.5)
+
```

Comment:
how about this line ``assert_almost_equal(lc.fold(period=1, phase=-0.1).time[0], 0.1)``?
Could you add it back, but obviously with the expected result according to your modifications?